### PR TITLE
Spanish translation for Beyond Agile article, by Autentia.

### PR DESCRIPTION
**Pull request by paniadri**
- Spanish translation for Beyond Agile article.
- Replaced some links for the alternatives in Spanish.
- It's been respected the way paragraphs are structured making easier for you to review the article.

Could it be possible to give <a href="http://autentia.com/">Autentia</a> credit for this translation? It would be very appreciated.

### CODE REVIEW

**Comment by ahmadnassri** [comment]
Thanks @adrianpelaez this is awesome, I'll figure out a new structure for the site to support translations, and yes will reference you and Autentia for credits :)

**Comment by adrianpelaez** [comment]
I'm glad you liked it! :)

**Comment by ahmadnassri** [comment]
keeping the PR open until I'm able to make the change (and thus keeping your contribution in the history as well)

**Comment by adrianpelaez** [comment]
sure, my bad, sorry ;)

File: opendevelopmentmethod-SpanishTranslation.txt [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+Las startups y negocios cuya mano de obra solía ser escasa y localizada están ahora más globalizadas, con empleados colaborando desde múltiples países y zonas horarias. Además de una nueva mano de obra, las tecnologías e idiomas de los que disponemos en nuestros lugares de trabajo desarrollan su propia evolución, permitiendo a los negocios a operar de formas nuevas y más enérgicas. Mientras el mundo entero continúa desarrollándose a nuestro alrededor tiene sentido que nuestras metodologías de desarrollo de software evolucionen también.
+
+¿POR QUÉ?
+
+El Open Development Method es un resumen de lecciones aprendidas en algunos de los grandes equipos punteros de Agile, Scrum, desarrollo en cascada y otras metodologías comunes.
+
+Después de presenciar esas tecnologías dejando de estar a la altura cuando alguna variable inesperada entraba en juego, especialmente cuando los equipos eran distribuidos, políglotas, y multifuncionales (un escenario que los equipos en startups actuales están experimentando), me he dado cuenta de que mientras la economía global continúa creciendo, los espacios de trabajo más tradicionales y centralizados ya no siguen siendo necesarios.
```
Comment (macuenca) [inline opendevelopmentmethod-SpanishTranslation.txt]: The last sentence of this paragraph in the original article reads `traditional notions of centralized workspaces are no longer desired`. The Spanish translation literally says `the more traditional and centralized workspaces are no longer needed.` which may sound better, but changes drastically the spirit of the original article.

I think this is an important assertion, and should not be changed in this case. I would suggest sticking to the original article verbatim, and changing this last sentence to: `las nociones tradicionales de espacios de trabajo centralizados ya no son el objetivo deseado.`

File: opendevelopmentmethod-SpanishTranslation.txt [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+LOS PRINCIPIOS DEL OPEN DEVELOPMENT METHOD
+
+Calidad del código.
+
+Dentro del open development method, la calidad de código es la reina. Deberías formularte las siguientes preguntas clave cada vez que escribes código:
+
+¿Este código es legible?
+¿Este código es testeable?
+¿Este código es modular?
+¿Este código es económico/rentable?
```
Comment (macuenca) [inline opendevelopmentmethod-SpanishTranslation.txt]: I would use *económico* leaving *rentable* out of the question. The author wants good code to be associated with exactly four adjectives. Is up to the reader to later figure out what exactly *economical* code entails.

File: opendevelopmentmethod-SpanishTranslation.txt [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+Democracia.
+
+Además de los debates abiertos y la compartición de ideas dentro de la comunidad de un proyecto, los miembros de esa comunidad deberían implicarse en la toma de decisiones. Cuando las decisiones se toman a puerta cerrada entre ejecutivos o managers, normalmente se tiene un punto de vista más limitado o escaso. Cuando todo el mundo está involucrado en las decisiones, incluyendo los usuarios o clientes, existe una probabilidad mayor de detectar y resolver problemas potenciales antes de que se conviertan en problemas.
+
+UNA INICIATIVA ABIERTA.
+
+En vez de permitir que una persona o un grupo pequeño de personas desarrolle esta metodología, el proceso debería ser tan abierto e inclusivo como cualquier otro proyecto open source.
+
+Es posible que haya prioridades o apartados que necesiten mejoras, así que por favor, únete al debate en GitHub, y trabajemos juntos en crear una lista de directrices que merezca la pena y permita triunfar a equipos de desarrollo de todos los tamaños.
```
Comment (macuenca) [inline opendevelopmentmethod-SpanishTranslation.txt]: Consider adding the link to https://github.com/OpenDevelopmentMethod/discussion in this last paragraph with a remark stating that it's written in English.

**Comment by macuenca** [comment]
As a native speaker I consider this a very good translation. Please consider my comments.
Good work:ok_hand:

**Comment by adrianpelaez** [comment]
Thanks for the suggestions @macuenca, updated considering your thoughts :)

**Comment by ahmadnassri** [comment]
@adrianpelaez @macuenca awesome! thanks for this guys! currently in London on a business trip, will be updating the site with this soon!

**Pull request closed without merging**